docs/VERIFY_OSCAL_CONTENT_QUALITY.md

```diff
@@ -0,0 +1,61 @@
+# Test for verify OSCAL content quality
+
+All discussions about this test can be found at [CPLYTM-254](https://issues.redhat.com/browse/CPLYTM-254) comment section. 
+
+## Test workflow 
+
+This test trigger by [github workflow](../.github/workflows/quality-test.yml), `pull request` event.
+
+What this workflow do:
+
+- Setup complyctl test environment
+- Get changed files of current pull request, filter files according to what we need to test.
+- Group changed files into `catalogs`, `profiles`, `component-definitions`
+- Calling [verify-OSCAL-contents.sh](../scripts/verify-OSCAL-contents.sh) with correct parameters to
+verify changed OSCAL files.
+
+Currently, we only have one job for testing `fedora cusp` related OSCAL content. Other operating
+system's OSCAL content can not be tested right now. But when other system's environment is ready, we can
+add more test jobs easily.
+
+## verify-OSCAL-contents.sh script
+
+This script verify OSCAL contents(CATALOG, PROFILE, COMPONENT DEFINITION) using complyctl and OpenSCAP,
+make sure OSCAL rules align with OpenSCAP rules.
+
+usage: 
+```shell
+./scripts/verify-OSCAL-contents.sh "$catalogs" "$profiles" "$component_definitions" "$product_name"
+
+# OSCAL Catalogs relative path string, multiple elements separate by space
+catalogs=$1
+# OSCAL Profiles relative path string, multiple elements separate by space
+profiles=$2
+# OSCAL Component Definitions relative path string, multiple elements separate by space
+component_definitions=$3
+# test product name 
+product=$4  
+```
+
+### What this script do:
+
+- Running complyctl `list`, `plan`, `generate` commands using changed OSCAL contents.
+Then calculate rule alignment between OSCAL and openscap, if alignment percentage lower than 60%(we can change this any time),
```

Comment:
```suggestion
Then calculate rule alignment between OSCAL and OpenSCAP, if alignment percentage lower than 60%,
```
We could remove include a dedicated sentence informing the 60% is a educated guess that can be reviewed and updated as we collect and analyse jobs results along the time.